core/templates/components/forms/custom-forms-directives/image-uploader.directive.ts

```diff
@@ -36,16 +37,19 @@ angular.module('oppia').directive('imageUploader', [
         onFileChanged: '=',
         errorMessage: '@',
         width: '@',
-        getAllowedImageFormats: '&allowedImageFormats'
+        getAllowedImageFormats: '&allowedImageFormats',
+        backgroundWhileUploading: '<'
       },
       templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
         '/components/forms/custom-forms-directives/' +
         'image-uploader.directive.html'),
       link: function(scope: ImageUploaderCustomScope, elt) {
         var onDragEnd = function(e) {
           e.preventDefault();
-          $('.image-uploader-drop-area').removeClass(
-            'image-uploader-is-active');
+          scope.backgroundWhileUploading = false;
+          // Since we are using JQlite for event handling
+          // we should trigger scope.$apply() Manually.
```

Comment:
This doesn't answer "why" we trigger a digest. You need to mention that it runs outside angular and that is why we have to manually trigger it.

Reply:
ditto below

Reply:
Done.